PROJECT

```diff
@@ -1,7 +1,10 @@
-domain: tmax.io
-repo: hypersds-operator
+domain: github.com
```

Comment:
It seems to be necessary to check if there is no association with the `apiVersion` of cr. Specifically, it is `group` of crd.